en/release-notes.md

```diff
@@ -118,256 +46,174 @@ within high-availability environments, the configuration of *bind* has been
+  This has now been consolidated into a single query per transaction.
+- Further reductions in the number of database queries have been made,
+  significantly cutting the queries made by the boot source cache image
+  import process from over 100 to just under 5.  These changes greatly improves
+  MAAS performance and database utilisation in larger environments. More
+  improvements will continue to be made as we continue to examine various areas
+  in MAAS.
 
-- On every transaction, MAAS performed 2 queries to update the timestamp. This
-  has now been consolidated into a single query per transaction.
-
-These changes greatly improves MAAS performance and database utilisation in
-larger environments. More improvements will continue to be made as we continue
-to examine various areas in MAAS.
-
-#### UI optimisation
+**UI optimisation**
 
-MAAS is being optimised to reduce the amount of data loaded in the WebSocket
+MAAS has been optimised to reduce the amount of data loaded in the WebSocket
 API to render the UI. This is targeted at only processing data for viewable
-information, improving various legacy areas. Currently, the work done in this
-area includes:
+information, improving various legacy areas. The work done in this area
+includes:
 
-- Script results are only loaded for viewable nodes in the machine listing
-  page, reducing the overall amount of data loaded.
+- Various web UI performance improvements include better filtering of node types
+  for machines, pods and zones.
+- Script results are only loaded for viewable nodes in the machine listing page,
+  reducing the overall amount of data loaded.
 - The node object is updated in the WebSocket only when something has changed
   in the database, reducing the data transferred to the clients as well as the
   amount of internal queries.
+- Only load historic script results (e.g. old commissioning/testing results) when
+  requested / accessed by the user, instead of always making them available over
+  the websocket.
+- Only load node objects in listing pages when the specific object type is
+  requested. For instance, only load machines when accessing the machines tab
+  instead of also loading devices and controllers.
+- Change the UI mechanism to only request OS Information only on initial page
+  load rather than every 10 seconds.
 
-#### Audit logging
+#### KVM pod improvements
 
-Continuing with the audit logging improvements, *alpha2* now adds audit logging
-for all user actions that affect Hardware Testing & Commissioning.
+KVM pods were initially created to help developers quickly iterate and test new
+functionality while developing MAAS. But KVM pods have also become an essential
+tool for administrators, allowing them to make better use of resources across
+the datacenter. As KVM pods were initially targeted at developers, MAAS lacked
+features more useful for administrators. MAAS 2.4 rectifies these shortcomings
```

Comment:
I feel this paragraph can be lightened up. I find there is too much repetition (developers, developing, administrators).